modules/layers/src/arc-layer/arc-layer-uniforms.ts

```diff
@@ -0,0 +1,39 @@
+import {ShaderModule} from '@luma.gl/shadertools';
+import {UniformTypes} from '@deck.gl/core';
+
+const uniformBlock = `\
+uniform arcUniforms {
+  bool greatCircle;
+  bool useShortestPath;
+  float numSegments;
+  float widthScale;
+  float widthMinPixels;
+  float widthMaxPixels;
+  highp int widthUnits;
+} arc;
+`;
+
+export type ArcProps = {
```

Comment:
Thought: To avoid confusion with other props types one might consider going with `ArcModuleProps` ?

Reply:
Here I would say we do what we've done in the past and review the API prior to 9.1 release. It's easy enough to rename across the codebase